styles.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --blue: #00acdc;
+}
+
+header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+header .logo img {
+  height: 40px;
+  width: 40px;
+}
```

Comment:
Fix it everywhere 
<img width="1099" alt="image" src="https://github.com/user-attachments/assets/598868e6-78ef-4c0c-9e46-d1b2d65b0b8a" />